assets/sass/patterns/molecules/teaser.scss

```diff
@@ -186,16 +186,6 @@
 
 }
 
-@media only all and (min-width: #{$bkpt-site--medium}px) {
-
-  .teaser__picture {
```

Comment:
Removing this block will affect all `.teaser__picture` instances, not just those appearing in the grid listing. Is this intended, and if so has it been cross browser tested in all teaser types?

A lighter touch would be to rather modify the selector so that it only doesn't apply to those images where a problem has been identified, e.g. something like `.teaser:not(:teaser--grid-style) .teaser__picture img` (untested).